_config.yml

```diff
@@ -0,0 +1,30 @@
+title: Meteor Demo
+author: AIOps team
+# logo: logo.png
```

Comment:
Can we add our logo here? 
![image](https://user-images.githubusercontent.com/4494906/133490551-28c5e4c9-c9d1-4758-a917-f646718b780b.png)